includes/Maps_MappingService.php

```diff
@@ -248,6 +248,17 @@ public function addResourceModules( $modules ) {
 	}
 
 	/**
+	 * Adds the layer dependencies.
+	 *
+	 * @param array $dependencies
+	 */
+	public function addLayerDependencies( array $dependencies ) {
```

Comment:
I could name it `addDependencyArray` because `addDependencies( &$parserOrOut )` already exists and does something different.